starters/next-wordpress-starter/lib/Posts.js

```diff
@@ -51,3 +51,27 @@ export async function getPostByUri(uri) {
 
 	return post;
 }
+
+export async function getPostPreview(id, credentials) {
+	client.setHeaders({ Authorization: `Basic ${credentials}` });
```

Comment:
Can we call `getAuthCredentials()` here instead of passing `credentials` in as an arg? Same question for `getPagesPreview`